cnames_active.js

```diff
@@ -213,6 +213,7 @@ var cnames_active = {
   "artalk": "artalk-demo.qwqaq.com", // noCF
   "artery": "arteryjs.github.io/gh-pages", // noCF? (don´t add this in a new PR)
   "arthurmbandeira": "arthurmbandeira.github.io",
+  "artificial": "fabiosmuu.github.io/artificial"
```

Comment:
```suggestion
  "artificial": "fabiosmuu.github.io/artificial",
```
Your entry needs a comma at the end of the line